README.md

```diff
@@ -32,10 +32,15 @@ We are a community of innovative Data Storage engineers that contribute to this
 This Community repository is the ```central location``` for project info and files like our governance, contribution, code of conduct, security and vision for the OpenEBS organization laid out by the Maintainers. You'll also find a [Top level Issues List](https://github.com/openebs/openebs/issues) for Organization related issues that span across products/projects. We also provide a [main Discussions forum](https://github.com/openebs/openebs/discussions) where **```everyone```** can collaborate and discuss anything.
 
 - Homepage: [openebs.io](https://openebs.io/)
-- Mailing list: openebs-team@googlegroups.com
-- Slack: [slack](https://kubernetes.slack.com/messages/openebs)
-- Twitter: [openebs](https://twitter.com/intent/follow?screen_name=openebs)
+- Maintainers' email: openebs-team@googlegroups.com
```

Comment:
I am not sure what's reason for not calling it mailing list, all other projects do so? This is supposed to the contact for the end users with the maintainers.

Also if you are changing these things make sure the change is done across all the sub-projects.

Reply:
I haven't seen another project call the contact email a mailing list. The mailing list is usually something you subscribe to and receive newsletters on, usually monthly.